src/Symfony/Bundle/FrameworkBundle/Tests/Routing/RedirectableUrlMatcherTest.php

```diff
@@ -38,26 +38,6 @@ public function testRedirectWhenNoSlash()
         );
     }
 
-    public function testSchemeRedirectBC()
```

Comment:
Wouldn't it be better to prefix the test method's name with `testLegacy` instead of removing it?